opi/plugins/packman.py

```diff
@@ -21,3 +21,6 @@ def run(cls, query):
 			'libavcodec-full',
 			'vlc-codecs',
 		])
+		# Install other codecs
+		opi.install_packages(['gstreamer-plugins-good',
```

Comment:
Could you make the formatting analog to the upper one?

```suggestion
		opi.install_packages([
			'gstreamer-plugins-good',
			'gstreamer-plugins-good-extra',
		])   
```